Description: Remove deprecated key and add Keywords key.
             GenericName  and Comment fields as asked by the upstream.
Author: Herbert Parentes Fortes Neto <hpfn@ig.com.br>
Last-Update: 2015-11-17
Index: webcamoid-6.2.0/webcamoid.desktop
===================================================================
--- webcamoid-6.2.0.orig/webcamoid.desktop
+++ webcamoid-6.2.0/webcamoid.desktop
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=Webcamoid
 GenericName=Webcam Capture Software
 GenericName[ca]=Programari de Captura de Càmera web
@@ -15,20 +14,21 @@ GenericName[pt]=Software de Captura de W
 GenericName[ru]=Веб-камера захвата программного обеспечения
 GenericName[zh_CN]=摄像头捕捉软件
 GenericName[zh_TW]=攝像頭捕捉軟件
-Comment=Webcam Capture Software
-Comment[ca]=Programari de Captura de Càmera web
-Comment[de]=Webcam-Capture-Software
-Comment[el]=κάμερα συλλαμβάνει το λογισμικό
-Comment[es]=Programa para Captura de la Webcam
-Comment[fr]=Logiciel de Capture Webcam
-Comment[gl]=Programa de Captura de Webcam
-Comment[it]=Webcam Capture Software
-Comment[ja]=ウェブカメラのキャプチャソフトウェア
-Comment[ko]=웹캠 캡처 소프트웨어
-Comment[pt]=Software de Captura de Webcam
-Comment[ru]=Веб-камера захвата программного обеспечения
-Comment[zh_CN]=摄像头捕捉软件
-Comment[zh_TW]=攝像頭捕捉軟件
+Comment=Take photos and record videos with your webcam
+Comment[ca]=Fer fotos i gravar vídeos amb la seva webcam
+Comment[de]=Maak foto's en video's opnemen met uw webcam
+Comment[el]=Τραβήξτε φωτογραφίες και εγγραφή βίντεο με την κάμερα σας
+Comment[es]=Tome fotos y grabe videos con su camara web
+Comment[fr]=Prenez des photos et enregistrer des vidéos avec votre webcam
+Comment[gl]=Facer fotos e gravar vídeos coa súa cámara web
+Comment[it]=Scatta foto e registrare video con la tua webcam
+Comment[ja]=ウェブカメラで写真や記録ビデオを撮影
+Comment[ko]=웹캠으로 사진과 기록 비디오를 촬영
+Comment[pt]=Tirar fotos e gravar vídeos com sua webcam
+Comment[ru]=Возьмите фотографии и записывать видео с веб-камеры
+Comment[zh_CN]=拍摄照片和录制视频与您的摄像头
+Comment[zh_TW]=拍攝照片和錄製視頻與您的攝像頭
+Keywords=photo;video;webcam;
 Exec=webcamoid
 Icon=camera-web
 Terminal=false
